plugins/pulp_rpm/plugins/migrations/0024_errata_pushcount_string.py

```diff
@@ -0,0 +1,33 @@
+"""
+This migration casts the pushcount field as a String.
+
+For the units_erratum collection, any object with the field 'pushcount' storing a non-String is
+cast to a String and saved. This is required since non String values will not validate with the
+Mongoengine definition.
+"""
+from pulp.server.db import connection
+
+
+def migrate(*args, **kwargs):
+    """
+    Perform the migration as described in this module's docblock.
+
+    :param args:   unused
+    :type  args:   list
+    :param kwargs: unused
+    :type  kwargs: dict
+    """
+    db = connection.get_database()
+    errata_collection = db['units_erratum']
+
+    for erratum in errata_collection.find():
+        changed = False
+        if not isinstance(erratum['pushcount'], basestring):
```

Comment:
This will fail with ``KeyError`` if the key doesn't exist in the document. Can we assume that all documents in this loop will have it? If not, the query suggested above is one way to do so.